sources/HUBContentOperationWrapper.m

```diff
@@ -56,29 +58,30 @@ - (void)performOperationForViewURI:(NSURL *)viewURI
                      previousError:(nullable NSError *)previousError
 {
     self.isExecuting = YES;
-    
+
+    id<HUBContentOperationContext> const context = ({
```

Comment:
What's the point of the `({ })`-scoping here? Haven't seen that in obj-c before 😮